### DISCUSSION TITLE
[Feature Request] clear context button

### DISCUSSION THREAD

**Discussion by GrayXu** [open-webui/open-webui#4134] [General]
**Is your feature request related to a problem? Please describe.**  
A common need is to discuss content unrelated to the previous user context under a given configuration (system prompt, model parameters). 

If a "clear context" button is added to the chat interface, we can further reduce wasted tokens and continue with the task without needing to recreate the conversation.   
(Under the current open-webui, if you want to recreate a conversation, you would still have to select multiple models again.)

**Describe the solution you'd like**  
Adding a clear context button, like in ChatGPTNextWeb/ChatGPT-Next-Web, would be a simple yet useful feature.

**Comment by kasxi** [comment]
Would this function kind of like a "clear chat' button so rather than having to create a new chat, you could just clear the conversation to date & start again? Because I have been thinking about a feature like this as well.

Throughout the day I have tons of narrow focus tasks that I don't want prior chat history to contaminate so I just start a new chat, being able to pin a single "general" chat & just clear the history when I want to focus on something new would save me a ton of tidy up at the end of the day getting rid of all the random chats I've created throughout the day.

**Reply by GrayXu** [reply]
My original goal is to keep the complex setup process for a conversation.  
So if there's a "clear chat history button" coupled with the existing "copy chat button", it can also meet my needs. This is also a solution!

**Comment by wyxogo** [comment]
I really hope this feature added to this project

**Reply by euroblaze** [reply]
@wyxogo 

- Is that not the same as starting a new chat?
- And delete the old one?

**Reply by GrayXu** [reply]
@euroblaze Here are the differences:
1. I can view the history within a conversation (suitable for simple workflows like translation, polish, etc., with no context required)
2. New chats cannot inherit any attributes from old conversations, including which models were selected and what the model parameter configurations were.
